pkg/master/master.go

```diff
@@ -212,11 +216,18 @@ type Master struct {
 	// External host is the name that should be used in external (public internet) URLs for this master
 	externalHost string
 	// clusterIP is the IP address of the master within the cluster.
-	clusterIP            net.IP
-	publicReadWritePort  int
-	serviceReadWriteIP   net.IP
-	serviceReadWritePort int
-	masterServices       *util.Runner
+	clusterIP           net.IP
+	publicReadWritePort int
+	serviceReadWriteIP  net.IP
+	masterServices      *util.Runner
+
+	// Ports to be exposed for the master service.
+	// servicePorts is injectable in the event that more ports
+	// (other than the default 443/tcp) are exposed on the master
```

Comment:
What this is still missing is why.  Why would more ports be exposed?  Why would they need to be load balanced by the master service?

Something like "...because this pkg is linked by out-of-tree projects like openshift, which want to use our master but also do more stuff."